design/KruizeConfiguration.md

```diff
@@ -139,3 +139,7 @@ The following environment variables are set using the `kubectl apply` command wi
     - Value: "true"
     - Details: This flag is added for getting the details of the inputs passed to the APIs and the corresponding response 
       generated by it. This helps us in debugging the API easily in case of failures.
+- **isROSEnabled**
+  - Description:  if set to True, the ROS application will use the old architecture, where ROS handles experiment creation, result updates, and recommendation updates. If set to False, ROS will utilize the Bulk API, allowing Kruize to manage experiment creation and generate recommendations on its behalf.
+    Default value is false.
+  - value: "false"
```

Comment:
This flag is not a either / or for Remote method or the local method. This flag should enable "Remote" method without having anything to do with the "Local" method.

```
**isROSEnabled**
- Description:  This flag enables the remote APIs such a updateResults and the corresponding DB Tables. If set to false, the corresponding APIs and the DB tables are not supported.
```

Reply:
This means that even if `isROSEnabled` is set to `true`, bulk APIs and corresponding DB tables are still accessible

Reply:
Yes Dino .. If it is false Im planning to disable API and db table creation.

Reply:
done